roster2/src/handlebars/members_header.handlebars

```diff
@@ -2,7 +2,7 @@
     <div class="roster-table-header-group">
         <div class="roster-table-row">
             {{#if viewPicture}}
-            <div class="roster-table-cell">{{translate 'facet_picture'}}</div>
+            <div class="roster-table-cell" id="roster-picture-header-cell">{{translate 'facet_picture'}}</div>
```

Comment:
does this id get repeated for each member? if yes, id should be unique

Reply:
Not repeated; this file defines the "header" row of the "table", which is only visible in spreadsheet/table/list view mode.

Reply:
👍